src/code/z_demo.c

```diff
@@ -1808,6 +1808,33 @@ void CutsceneCmd_Text(PlayState* play, CutsceneContext* csCtx, CsCmdText* cmd) {
     }
 }
 
+#ifdef ENABLE_MOTION_BLUR
+void CutsceneCmd_MotionBlur(PlayState* play, CutsceneContext* csCtx, CsCmdMotionBlur* cmd) {
+    if ((csCtx->curFrame >= cmd->startFrame) && (cmd->endFrame >= csCtx->curFrame)) {
+        f32 lerp = Environment_LerpWeight(cmd->endFrame, cmd->startFrame, csCtx->curFrame);
+
+#ifdef ENABLE_MOTION_BLUR_DEBUG
+        osSyncPrintf("originalBlurAlpha: 0x%X, lerp: %f\n", play->csCtx.originalBlurAlpha, lerp);
+#endif
+
+        if (cmd->type == CS_MOTION_BLUR_ENABLE) {
+            if (csCtx->originalBlurAlpha == 0) {
+                csCtx->originalBlurAlpha = cmd->alpha;
+            }
+
+            Play_EnableMotionBlur(lerp * cmd->alpha);
```

Comment:
Would be good to test a `startFrame` and `endFrame` that are far apart to make sure the "gradual" change in motion blur works